heron/tools/cli/src/python/response.py

```diff
@@ -33,14 +34,17 @@ class Status(object):
   Ok = 0
   InvocationError = 1
   HeronError = 100
+  DryRun = 200
 
 def status_type(status_code):
   if status_code == 0:
     return Status.Ok
   elif status_code < 100:
     return Status.InvocationError
-  else:
+  elif status_code >= 100 and status_code < 200:
```

Comment:
As written you're defining anything >= 200 returned from the java process to be represented as a dry run by the python process. Why?

I thought you were adding a new 200 code explicitly for dry run, which is why I suggested changing the logic from this on master:
```
else:
  return Status.HeronError
```
to this:
```
elif status_code == 200:
  return Status.DryRun
else:
  return Status.HeronError
```